data-prepper-api/src/test/java/org/opensearch/dataprepper/typeconverter/LongConverterTests.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ package org.opensearch.dataprepper.typeconverter;
+
+ import org.junit.jupiter.api.Test;
+ 
+ import static org.hamcrest.CoreMatchers.equalTo;
+ import static org.hamcrest.MatcherAssert.assertThat;
+ import static org.junit.jupiter.api.Assertions.assertThrows;
+ 
+ public class LongConverterTests {
+     @Test
+     void testStringToLongConversion() {
+         LongConverter converter = new LongConverter();
+         final String stringConstant = "1234";
+         assertThat(converter.convert(stringConstant), equalTo(Long.parseLong(stringConstant)));
+     }
+     @Test
+     void testfloatToLongConversion() {
+         LongConverter converter = new LongConverter();
+         final Float floatConstant = (float)1234.56789;
+         assertThat(converter.convert(floatConstant), equalTo((long)(float)floatConstant));
+     }
+     @Test
+     void testDoubleToLongConversion() {
+        LongConverter converter = new LongConverter();
+        final Double doubleConstant = (double)12345678.12345678;
```

Comment:
It may be good to have a test that converts a value which is larger than the maximum integer in addition to this one. You could make a large double constant like `2.0 * Integer.MAX_INT`.

Reply:
Add a test case `testDoubleToLongConversionWithBigValue` for it.